### PR TITLE
Add BitcrowdEcto.Random

### PR DESCRIPTION
**Pull request by maltoe**

### CODE REVIEW

**Review by agatheblues — COMMENTED** [review]
Thanks for doing this! I think we could list the new module in the README then 📝

File: lib/bitcrowd_ecto/random.ex [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  @doc """
+  Generates a random token suitable for inclusion in URLs.
+  """
+  @doc since: "0.6.0"
+  @spec url_token() :: String.t()
+  @spec url_token(bytes :: non_neg_integer) :: String.t()
+  def url_token(bytes \\ 32) do
+    bytes
+    |> random_bytes()
+    |> Base.url_encode64(case: :lower, padding: true)
```
[the remark this answers is not in the document]
Comment (maltoe) [reply lib/bitcrowd_ecto/random.ex:24]: 😄  uuh how it just became obvious from where I got the code

File: lib/bitcrowd_ecto/random.ex [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  @spec uuid() :: String.t()
+  defdelegate uuid, to: Ecto.UUID, as: :generate
+
+  @doc """
+  Generates a random token suitable for inclusion in URLs.
+  """
+  @doc since: "0.6.0"
+  @spec url_token() :: String.t()
+  @spec url_token(bytes :: non_neg_integer) :: String.t()
+  def url_token(bytes \\ 32) do
```
[the remark this answers is not in the document]
Comment (maltoe) [reply lib/bitcrowd_ecto/random.ex:21]: 🤔  i found the tokens to be surprisingly short. But maybe we should keep the default.

[the remark this answers is not in the document]
Comment (maltoe) [reply lib/bitcrowd_ecto/random.ex:21]: not sure if it needs a migration guide, or whether i'm willing to write one :)

**Comment by maltoe** [comment]
@agatheblues updated

**Review by agatheblues — APPROVED** [review]
🚢 💨

**Pull request merged**